application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -1056,66 +544,8 @@ define('diagram-link-editor', [
           // Keep only the text content.
           str = $('&lt;div/&gt;').html(str).text() || this.foAltText;
         }
-        // Update links inside foreignObject to use absolute URLs, since the generated SVG needs to be portable.
-        $(fo).find('a').each(function() {
-          var oldLink = $(this).attr('href');
-          if (diagramLinkHandler.isXWikiCustomLink(oldLink)) {
-            var newLink = diagramLinkHandler.getURLFromCustomLink(oldLink);
-            $(this).attr('href', newLink);
-          }
-        });
```

Comment:
Are the links of the new PDF export absolute?

Reply:
After some testing, it appears this code is never called, and the links in the XWiki export are not displayed at all. Do you think I should remove this code now, or wait until I actually fix the issue?